### PR TITLE
Add warnings for negative critical time

### PR DESCRIPTION
**Pull request by sergioc**

### CODE REVIEW

**Comment by aleksandr-samila** [comment]
Some plumbing was added

**Comment by aleksandr-samila** [comment]
Updated the sources to highlight few more negative numbers. Should be ready for the final review

File: src/ServicePulse.Host/app/js/app.constants.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 window.defaultConfig = {
     default_route: '/dashboard',
     version: '1.2.0',
-    service_control_url: 'http://localhost:33333/api/',
-    monitoring_urls: ['http://localhost:33633/'],
+    service_control_url: 'http://localhost:49200/api/',
```
Comment (WojcikMike) [inline src/ServicePulse.Host/app/js/app.constants.js:4]: This change should be reverted

**Review by WojcikMike — APPROVED** [review]
Just reverse the change to the app.contants.js. Then you can merge it

**Pull request merged**